sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -15,7 +15,9 @@
 # limitations under the License.
 #
 
-"""Dataflow client utility functions."""
+""" For internal use only. No backwards compatibility guarantees.
```

Comment:
Is it enough to just add this comment here and not add the same comment to individual classes in this file?

For example iobase.py only comment is added to the top.